server/routes/appointments/appointmentsController.test.ts

```diff
@@ -209,9 +213,9 @@ describe('Scheduling a supplier assessment appointment', () => {
           .post(`/service-provider/referrals/${referral.id}/supplier-assessment/schedule/${draftBooking.id}/details`)
           .type('form')
           .send({
-            'date-day': '24',
-            'date-month': '3',
-            'date-year': '2025',
+            'date-day': dateOfAppointment.getUTCDate(),
```

Comment:
can we keep the test data as hard-coded values - having the values calculated like this can be confusing when looking at this test as you have to go to the referral factory to read what the date is actually being set as

Reply:
Yep, this is no longer needed now that the mocked referral doesn't use new Date() anymore. It has been hardcoded to 1st Jan 2022 so we're safe to hard-code these as well